CMakeLists.txt

```diff
@@ -219,7 +217,6 @@ build_third_party_lib(sqlite3 sqlite3)
 
 # build tree-sitter parser
 include_directories(third-party/tree-sitter/tree-sitter/lib/include)
-include_directories(third-party/tree-sitter/tree-sitter-opengoal/include)
```

Comment:
similarly curious, not actually needed?

Reply:
That path doesn't exits, so I was getting spammed with warnings for it.